[SAASINT-4032] DDS: DNSFilter: Crawler Integration v1.0.0 (#20384)
DataDog/integrations-core: dnsfilter/README.md

# DNSFilter

## Overview

> [DNSFilter][1] is a cloud-based content filtering and threat protection by blocking internet threats at the DNS layer. It protects organizations by preventing access to malicious domains, phishing sites, and other cyber threats, ensuring a safer internet experience.

Reviewer: ```suggestion
[DNSFilter][1] is a cloud-based content filtering tool that blocks internet threats at the DNS layer. It protects organizations by preventing access to malicious domains, phishing sites, and other cyber threats, ensuring a safer internet experience.
```
The first sentence is not grammatically correct. I made one possible edit, but feel free to revise to your preferences.

This integration ingests the following logs:

- DNS Traffic Logs: Represents information about allowed and blocked DNS requests, threats, domains accessed, policies, networks, and other DNS-related traffic data.

> This integration seamlessly collects DNS Traffic Logs, channeling them into Datadog for analysis. Leveraging the built-in logs pipeline, these logs are parsed and enriched, enabling effortless search and analysis. The integration provides insight into dns traffic logs through out-of-the-box dashboards and includes ready-to-use Cloud SIEM detection rules for improved monitoring and security.

Reviewer: ```suggestion
This integration collects DNS Traffic Logs, channeling them into Datadog for analysis. Leveraging the built-in logs pipeline, these logs are parsed and enriched, enabling search and analysis. The integration provides insight into DNS traffic logs through out-of-the-box dashboards and includes ready-to-use Cloud SIEM detection rules for improved monitoring and security.
```

## Setup

> ### Generate API Credentials in DNSFilter

Reviewer: ```suggestion
### Generate API credentials in DNSFilter
```

Author: Have updated accordingly.

1. Login to the **DNSFilter dashboard** and navigate to **Account**.
2. Select **Account Settings**.

> 3. Navigate to **Security** tab.
> 4. Navigate to **API Keys** section, then click **CREATE KEY**.
> 5. Enter a key **Name** and select an **Expiration**.
> 6. Click **GENERATE KEY**.
> 7. Fetch API Key from **Your API Key** Section.

Reviewer: ```suggestion
3. Navigate to the **Security** tab.
4. Navigate to the **API Keys** section, then click **CREATE KEY**.
5. Enter a key **Name** and select an **Expiration**.
6. Click **GENERATE KEY**.
7. Fetch the API Key from the **Your API Key** Section.
```

Author: Have updated accordingly.

> ### Connect your DNSFilter Account to Datadog

Reviewer: ```suggestion
### Connect your DNSFilter account to Datadog
```

1. Add your API Key.

    | Parameters                            | Description                                                  |
    | ------------------------------------- | ------------------------------------------------------------ |

>     | API Key                               | The API Key of your DNSFilter Platform                       |

Reviewer: ```suggestion
    | API Key                               | The API Key of your DNSFilter platform                       |
```

    
2. Click the **Save** button to save your settings.

## Data Collected

### Logs

> The DNSFilter integration collects and forwards dns traffic logs to Datadog.

Reviewer: ```suggestion
The DNSFilter integration collects and forwards DNS traffic logs to Datadog.
```

Author: Have updated accordingly.
